fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/BaseProvider.java

```diff
@@ -76,6 +76,11 @@
     }
   }
 
+  @Override
+  public Set<R> addItem(R item) {
```

Comment:
Is this method required to compile? If not, I'd suggest removing, since it doesn't do what the method name suggests.

Reply:
It has to be there to compile.